app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -533,20 +536,55 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_selectAdminNavItem_adminControlsIsDisplayed() {
```

Comment:
Selecting the menu item again should not cause a crash. 
1. selecting admin control nav item open `AdministratorControlsActivity `, tested in the navigation test file.
2. clicking on the item again, shows `AdministratorControlsActivity`. 
So, these cases are covered and tested in both these test cases.